jsonschema-core.xml

```diff
@@ -2777,22 +2776,20 @@
                         </artwork>
                     </figure>
                     <t>
-                    Note that this pointer may not be resolvable by the normal JSON Pointer process
-                    due to the inclusion of these by-reference applicator keywords.
+                        Note that this pointer may not be resolvable by the normal JSON Pointer process
+                        due to the inclusion of these by-reference applicator keywords.
```

Comment:
If we talk about the evaluation paths as a transient document, and `evaluationPath` as pointers for that document, then we can drop notes like this.  We may want to talk about that in the section on dynamic scopes rather than here, so that it is defined earlier in the spec.

Reply:
I'll leave this here until it gets moved there.